demo/src/modules/home/containers/TableContainer.js

```diff
@@ -249,33 +249,25 @@ export default function (props) {
 		}
 	}
 
-	const customDropdownButton = {
-		text: "Custom drop",
-		color: "success",
-		header: "header",
-		items: [
-			{
-				text: "First item",
-				props: {
-					onClick: () => alert("First item was clicked!"),
-					className: "first-item-classname"
-				}
-				
-			},
-			{
-				text: "Second item",
-				props: {
-					onClick: () => alert("Second item was clicked!"),
-					className: "second-item-classname",
-					style: {
-						backgroundColor: "black",
-						color: "white"
-					}
-				}
-			}
-		]
+	const nextPage = () => {
+		if (page*limit < count) {
+			setPage(prev => prev+1)
+		}
+		else {
+			alert("You're on the last page");
```

Comment:
I think it would be better not to alert, but to disable button

Reply:
That's just demo in order to show how developer can use this `customComponent`. I suppose that's not needed to write there some real styling logic

Reply:
I think this would make code more complex and if someone will take a look on it in order to understand how to use it, then he may be a bit confused with all that stuff.